it/compare-generation.ps1

```diff
@@ -81,8 +81,10 @@ Get-FileHash -InputStream ([IO.MemoryStream]::new([char[]]$HashString1))
 $HashString2 = (Get-ChildItem $tmpFolder2 -Recurse | where { ! $_.PSIsContainer } | Get-FileHash -Algorithm MD5).Hash | Out-String
 Get-FileHash -InputStream ([IO.MemoryStream]::new([char[]]$HashString2))
 
-Write-Output "Folder 1: $tmpFolder1"
-Write-Output "Folder 2: $tmpFolder2"
+Write-Output "Folder Old: $tmpFolder1"
+Write-Output "Folder New: $tmpFolder2"
+
+diff --color -r "$tmpFolder1" "$tmpFolder2"
```

Comment:
this is not native to windows, powershell aliases it to compare-object but I don't thinnk it accepts the color option, maybe surround with a check for the OS?

Reply:
to be removed, sure 👍